app/src/main/res/values-hi-rIN/strings.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">नोट क्रिप्ट प्रो</string>
```

Comment:
Please do not translate the app name here, and also where "Note Crypt" is mentioned in the other strings